assets/js/wpmenucart-wc-block-support.js

```diff
@@ -0,0 +1,48 @@
+jQuery( function( $ ) {
+    let wpmenucart_ajax_timer;
+    function WPMenucart_Load_AJAX() {
+        let data = {
+            security:   wpmenucart_cart_ajax.nonce,
+            action:     "wpmenucart_ajax",
+        };
+
+        xhr = jQuery.ajax({
+            type:       'POST',
+            url:        wpmenucart_cart_ajax.ajaxurl,
+            data:       data,
+            success:    function( response ) {
+                jQuery('.wpmenucartli').html( response );
+            }
+        });
+    }
+    wp.hooks.addAction(
+        `experimental__woocommerce_blocks-cart-set-item-quantity`,
```

Comment:
The single quote should be `'`.

Reply:
Fixed cart page issue and updated the code.